src/state/stake/stake-constants.ts

```diff
@@ -279,8 +290,51 @@ const AURORA_POOLS: StakingTri[] = [
     rewardRate: dummyAmount,
     apr: 0,
     apr2: 0,
-    chefVersion: ChefVersions.V2
+    chefVersion: ChefVersions.V2,
+    doubleRewards: true
   },
+  {
+    ID: 9,
+    poolId: 2,
+    tokens: [WNEAR[ChainId.AURORA], LUNA[ChainId.AURORA]],
+    stakingRewardAddress: MASTERCHEF_ADDRESS_V2[ChainId.AURORA],
+    lpAddress: '0xd1654a7713617d41A8C9530Fb9B948d00e162194',
```

Comment:
this is LP address of already active pool: https://github.com/trisolaris-labs/apr/blob/master/apr.py#L95